src/pages/billing/CreateAzureBillingProjectModal.test.js

```diff
@@ -162,8 +162,8 @@ describe('CreateAzureBillingProjectModal', () => {
           listAzureManagedApplications: () => Promise.resolve(
             {
               managedApps: [
-                { applicationDeploymentName: 'testApp1', tenantId: 'fakeTenant1', subscriptionId: 'fakeSub1', managedResourceGroupId: 'fakeMrg1' },
-                { applicationDeploymentName: appName, tenantId: tenant, subscriptionId: subscription, managedResourceGroupId: mrg }
+                { applicationDeploymentName: 'testApp1', tenantId: 'fakeTenant1', subscriptionId: 'fakeSub1', managedResourceGroupId: 'fakeMrg1', assigned: true },
```

Comment:
In practice, `assigned` would be false correct?

Reply:
Yeah, I don't know why I made this true